### PR TITLE
add: vim replacement inside visual selection

### PR DESCRIPTION
**Pull request by cckcoder**
add one more tip for replacement in vim

### CODE REVIEW

File: content/programming-cookbook/vim-and-neovim/replacement-inside-visual-selection.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  console.log(key, value);
+}
+```
+
+## How to do with vim?
+
+### Steps
+
+- `shift + v` select what ever line do you want to
+- then `:s/let/var/g <CR>`
```
Comment (mildronize) [inline content/programming-cookbook/vim-and-neovim/replacement-inside-visual-selection.md:25]: I think you should explain how that command works.

File: content/programming-cookbook/vim-and-neovim/replacement-inside-visual-selection.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: replacement inside visual selection
```
Comment (mildronize) [inline content/programming-cookbook/vim-and-neovim/replacement-inside-visual-selection.md]: Please use Big Letter => "Replacement inside visual selection"

**Review by mildronize — CHANGES_REQUESTED** [review]
Good PR. Please change following my suggestion. 

Thank you for your contribution.

**Comment by aaronamm** [comment]
@mildronize Thank you so much for reviewing this PR.

**Review by aaronamm — APPROVED** [review]
Thank you for useful content.

**Pull request merged**